src/main/java/lotto/enums/Rank.java

```diff
@@ -0,0 +1,34 @@
+package lotto.enums;
+
+import java.util.Arrays;
+
+public enum Rank {
+	NOTHING(0L, 0L),
+	FOURTH(3L, 5000L),
+	THIRD(4L, 50000L),
+	SECOND(5L, 1500000L),
+	FIRST(6L, 2000000000L);
```

Comment:
```suggestion
	NOTHING(0L, 0L),
	FOURTH(3L, 5_000L),
	THIRD(4L, 50_000L),
	SECOND(5L, 1_500_000L),
	FIRST(6L, 2_000_000_000L);
```
이렇게 표현하면 숫자를 읽을 때 더 수월하답니다!